app/app.go

```diff
@@ -971,6 +972,28 @@ func NewApp(
 	return app
 }
 
+func (app *App) RegisterServices(cfg module.Configurator) {
+	// Register services from all unmodified modules
+	for _, module := range app.mm.Modules {
+		// skip registration of distribution services
+		if module.Name() == distrtypes.ModuleName {
+			continue
+		}
+		module.RegisterServices(cfg)
+	}
+
+	// register ditribution services except query server
+	distrtypes.RegisterMsgServer(cfg.MsgServer(), distrkeeper.NewMsgServerImpl(app.distrKeeper))
+	cfg.RegisterMigration(
```

Comment:
we should add a note here that this needs to be updated whenever we update cosmos SDK since new migrations may be updated.